test/test_extract.py

```diff
@@ -118,14 +118,11 @@ def test_extract_file_rpm(self):
                 for e in self.extractor.file_extractors[self.extractor.extract_file_rpm]
             ]
         ):
-            if sys.platform.startswith("linux"):
+            print(os.path.join(extracted_path, "usr", "bin", "curl"))
+            if True:
```

Comment:
We should be able to skip this line and put the indent of the next section up one level.